### PR TITLE
shrshinde/isSupported-remoteCamera-check

### PR DESCRIPTION
**Pull request by shrshindeMSFT**
- Adding isSupported check to remoteCamera capability.
- Adding unit test cases.

### CODE REVIEW

File: packages/teams-js/test/private/remoteCamera.spec.ts [hunk 1]
```diff
@@ -61,6 +62,23 @@ describe('remoteCamera', () => {
 
     Object.values(FrameContexts).forEach(context => {
       if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+        it(`remoteCamera.getCapableParticipants should throw error when remoteCamera is not supported. context : ${context}`, async () => {
```
Comment (erinha) [inline packages/teams-js/test/private/remoteCamera.spec.ts:65]: I think there should be at least one test  case where isSupported returns true and we set the runtimeConfig explicitly. Can you please add one if it's not present?

File: packages/teams-js/test/private/remoteCamera.spec.ts [hunk 2]
```diff
@@ -2,8 +2,9 @@ import { Utils } from '../utils';
 import { remoteCamera } from '../../src/private/remoteCamera';
 import { app } from '../../src/public/app';
 import { SdkError } from '../../src/public/interfaces';
-import { FrameContexts } from '../../src/public';
+import { call, FrameContexts } from '../../src/public';
```
Comment (erinha) [inline packages/teams-js/test/private/remoteCamera.spec.ts:5]: Typo? Don't see  `call` being used anywhere here.

**Review by erinha — CHANGES_REQUESTED** [review]
Looks good overall, one small comment.

**Comment by shrshindeMSFT** [comment]
I'll make those changes and push it.

File: packages/teams-js/test/private/remoteCamera.spec.ts [hunk 3]
```diff
@@ -375,6 +440,21 @@ describe('remoteCamera', () => {
 
     Object.values(FrameContexts).forEach(context => {
       if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+        it(`remoteCamera.registerOnCapableParticipantsChangeHandler should throw error when remoteCamera is not supported. context : ${context}`, async () => {
+          await utils.initializeWithContext(context);
+          utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
+          let CapableParticipants: remoteCamera.Participant[];
```
Comment (erinha) [inline packages/teams-js/test/private/remoteCamera.spec.ts:446]: nit: lower case "c" in "capableParticipants" for consistency.

**Review by erinha — CHANGES_REQUESTED** [review]
Sorry  if I missed the small comment about variable naming in the tests before.

File: packages/teams-js/test/private/remoteCamera.spec.ts (hunk 3 above)
Comment (shrshindeMSFT) [reply packages/teams-js/test/private/remoteCamera.spec.ts]: Fixed in the latest commit.

**Review by erinha — APPROVED** [review]
Looks good, thanks!

**Review by aeunms — APPROVED** [review]
🚀

**Pull request merged**